tox.ini

```diff
@@ -22,16 +22,7 @@ commands =
 passenv =
   ENVVAR_PREFIX_FOR_DYNACONF
   HOME
-  OS_AUTH_URL
-  OS_AUTH_VERSION
   OS_CLOUD
```

Comment:
I think we still want OS_*. openstacksdk should still use envvars if there is no cloud.yaml.

Reply:
Done